calc/pkg/package.json

```diff
@@ -18,4 +18,4 @@
   "main": "calc.js",
   "homepage": "https://github.com/paritytech/substrate-api-sidecar/tree/master/calc#readme",
   "types": "calc.d.ts"
-}
+}
```

Comment:
One last nit could you add the space here. When the wasm compiles each time it trims this whitespace.. Sorry abou t that thanks :)

Reply:
yes, thanks!